### PR TITLE
Testing cases and cleaning up

### PR DESCRIPTION
**Pull request by jonmc12**
The PR normalizes a Client, Service, Test pattern for each module: Podcast API, Supabase, and Transcriber.

Each class has had ChatGPT's opinions integrated for readability. Black is used to format syntax and Ruff is used to order inputs.

.env was removed for security. A .env.sample is included.

### CODE REVIEW

**Review by cts660 — APPROVED** [review]
LGTM

File: podcast_agent/services/transcriber.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    def transcribe_and_upload(self):
+        """
+        Transcribe an mp3 from a ListenNotes URL and upload to Supabase
+        """
+        podcast_api_service = PodcastApiService()
+        loop = asyncio.get_event_loop()
+        # transcription = self.transcribe_from_url(url=url)
+        episode_response = loop.run_until_complete(podcast_api_service.get_episode(
+            id="87a935f22138437baa4b0a1043895e9b",
+            show_transcript=1
```
Comment (cts660) [inline podcast_agent/services/transcriber.py:46]: should this `id` be an argument to `transcribe_and_upload`?

**Review by cts660 — APPROVED** [review]
LGTM, just had one question

File: podcast_agent/services/transcriber.py (hunk 1 above)
Comment (jonmc12) [reply podcast_agent/services/transcriber.py:46]: Yes, for sure, this is WIP - I'll work on it next.

**Pull request merged**